style.css

```diff
@@ -6,651 +6,1075 @@
-  align-items: center;
-  color: #ff6b00;
-  font-family: "Crete Round", sans-serif;
-  font-weight: 400;
-  font-size: 72px;
-  line-height: 88px;
-  letter-spacing: 0.37px;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-}
+  .card2-button {
+    background: #ff6b00;
+    letter-spacing: 0.03em;
+    color: #fff;
+    padding: 17px;
+    width: 329px;
+    flex: none;
+    order: 3;
+    flex-grow: 0;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    border: none;
+  }
 
-.about p {
-  display: flex;
-  flex-direction: row;
-  justify-content: center;
-  color: #f4f5f7;
-  font-family: "Inter", sans-serif;
-  font-weight: 400;
-  font-size: 16px;
-  line-height: 24px;
-  flex: none;
-  order: 1;
-  flex-grow: 0;
-}
+  .card-button:hover {
+    background-color: #ff6b00;
+    border: 1px solid white;
+  }
 
-.about-button {
-  width: 156px;
-  height: 48px;
-  background: #ff6b00;
-  flex: none;
-  order: 2;
-  flex-grow: 0;
-  border: none;
-  font-weight: 700;
-  font-size: 17px;
-  letter-spacing: 0.03em;
-  color: #fff;
-}
+  .card-button:active {
+    background-color: #ff6b00;
+    border: 1px solid white;
+  }
 
-.about-button:hover {
-  background-color: #ff6b00 !important;
-  border: 1px solid white !important;
-}
+  /* about section */
+  .about {
+    display: grid;
+    background-image: url(Images/Group1.png);
+    background-color: #1c1a19;
+    justify-content: center;
+    text-align: left;
+    padding: 100px 55px;
+    gap: 24px;
+    background-position: right bottom;
+    background-size: 140px;
+    background-repeat: no-repeat;
+    position: relative;
+    margin-top: 100px;
+    padding-bottom: 200px;
+  }
 
-.about-button:active {
-  background-color: #ff6b00 !important;
-  border: 1px solid white !important;
-}
+  .about h2 {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    color: #ff6b00;
+    font-family: "Crete Round", sans-serif;
+    font-weight: 400;
+    font-size: 60px;
+    line-height: 88px;
+    letter-spacing: 0.37px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
 
-/* skills section */
-.skill {
-  display: grid;
-  justify-content: center;
-  gap: 24px;
-  background: #1c1a19;
-  order: 3;
-  color: #fff;
-  padding-bottom: 79px;
-}
+  .about p {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    color: #f4f5f7;
+    font-family: "Inter", sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
 
-.skill-list {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  gap: 24px;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-  margin-top: 100px;
-}
+  .about-button {
+    width: 156px;
+    height: 48px;
+    background: #ff6b00;
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+    border: none;
+    font-weight: 700;
+    font-size: 17px;
+    letter-spacing: 0.03em;
+    color: #fff;
+  }
```

Comment:
- [ ] Your work on the desktop version is outstanding! :+1: However, to proceed to the next stage of the project, there are a few issues that I kindly request you address. These issues are as follows:
- There is a border on your button that is not present in the Figma template. Kindly remove this border to ensure that the button matches the design accurately.
- The separator on your page needs to take up more space of the page width to match the design in the Figma template more closely. Making this adjustment will help ensure consistency in the design and improve the overall appearance of the page.

**Your design**

![Screenshot from 2023-04-05 08-29-56](https://user-images.githubusercontent.com/105117832/230095432-bfbf535f-05a1-4a21-923d-18981589b169.png)

**Figma**

![Screenshot from 2023-04-05 08-26-59](https://user-images.githubusercontent.com/105117832/230095501-de73019b-86e7-4ed5-8a35-ebeaac52f2fd.png)